plugins/image2/plugin.js

```diff
@@ -932,6 +938,10 @@
 
 				// Image can be wrapped in link <a><img/></a>.
 				image = el.getFirst( 'img' ) || el.getFirst( 'a' ).getFirst( 'img' );
+
+				el.lockedStyle = el.lockedStyle || {};
```

Comment:
Shouldn't it be bound to the widget, not the element itself?

Reply:
At this point widget hasn't been initialized and there is no instance to which I could bind this attribute.

Reply:
But upcast method takes `data` as a second parameter. These locked styles could be treated as a part of widget's data.